docs/language/syntax.rst

```diff
@@ -44,8 +44,8 @@ languages. A bracket string begins with ``#[FOO[`` and ends with ``]FOO]``,
 where ``FOO`` is any string not containing ``[`` or ``]``, including the empty
 string. For example::
 
-   => (print #[["That's very kind of yuo [sic]" Tom wrote back.]])
-   "That's very kind of yuo [sic]" Tom wrote back.
+   => (print #[["That's very kind of you [sic]" Tom wrote back.]])
+   "That's very kind of you [sic]" Tom wrote back.
```

Comment:
I made that typo on purpose, so I could write "[sic]", so I could get square brackets into the string.